build/ci.go

```diff
@@ -227,6 +227,9 @@ func doTest(cmdline []string) {
 
 	// Run the actual tests.
 	gotest := goTool("test")
+	// Test a single package at a time. CI builders are slow
+	// and some tests run into timeouts under load.
+	gotest.Args = append(gotest.Args, "-p", "1")
```

Comment:
I have started modifying this file to support our build process.  This particular change is of no consequence for us